cmd/access/main.go

```diff
@@ -92,6 +93,7 @@ func main() {
 			flags.BoolVar(&retryEnabled, "retry-enabled", false, "whether to enable the retry mechanism at the access node level")
 			flags.BoolVar(&rpcMetricsEnabled, "rpc-metrics-enabled", false, "whether to enable the rpc metrics")
 			flags.StringVarP(&nodeInfoFile, "node-info-file", "", "", "full path to a json file which provides more details about nodes when reporting its reachability metrics")
+			flags.StringToIntVar(&apiRatelimits, "rate-limits", nil, "per second rate limits for Access API methods e.g. Ping=300,GetTransaction=500 etc.")
```

Comment:
@jordanschalm thanks for comment earlier. Added this new command line arg to read in the limits